renovate.json

```diff
@@ -1,23 +1,27 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "config:base"
-  ],
+  "extends": ["config:base"],
   "schedule": ["every weekend"],
   "packageRules": [
     {
       "matchManagers": ["github-actions"],
-      "groupName": "github-actions",
-      "commitMessageTopic": "github-actions"
+      "groupName": "github-actions"
     },
     {
-      "extends": "group:nodeJs",
-      "matchUpdateTypes": [
-        "minor",
-        "patch"
-      ],
-      "groupName": "nodeJs",
-      "commitMessageTopic": "NodeJs"
+      "matchManagers": ["gomod"],
+      "matchUpdateTypes": ["minor", "patch"],
+      "groupName": "go"
+    },
+    {
+      "matchManagers": ["npm"],
+      "matchDepTypes": ["dependencies"],
```

Comment:
NOTE: I'm not 100% sure about this depType since there don't seem to be any docs on available depTypes in renovate. I'm making a guess at this given other examples in the docs and npm terminology.

Reply:
@rarkins could you advise? Thanks!

Reply:
`depTypes` varies per manager. For some managers it can be dynamic from the user too. I the case of `npm`, `dependencies` and `devDependencies` are static terms and correct, as you've done

Reply:
Thank you!